client/src/components/common/TeacherTable/index.css

```diff
@@ -1,45 +1,46 @@
 .studentTable {
```

Comment:
I think we shouldn't use CamelCase in css
.student-table

most of classes name camelCase